apps/web/src/components/Profile/components/LanguageProfile/utils.ts

```diff
@@ -95,17 +95,22 @@ export const dataToFormValues = (data: PartialUser): FormValues => {
 
 export const getLabels = (intl: IntlShape) => ({
   consideredPositionLanguages: intl.formatMessage({
-    defaultMessage: "Language of positions you would like to be considered for",
-    id: "D1ZXOz",
+    defaultMessage: "Language of positions they'd like to be considered for",
+    id: "96w8Lm",
     description:
       "Legend for considered position languages check list in language information form",
   }),
-  firstOfficialLanguage: intl.formatMessage({
+  yourFirstOfficialLang: intl.formatMessage({
```

Comment:
I don't think you can do this, need two separate ones as it is reused in different contexts

![image](https://github.com/user-attachments/assets/a5d2c3db-32a0-4496-b9bd-f31f960d7655)

Reply:
Oops :laughing: thanks! Added back in [Add back considered language message](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/6ea2e5bdbbbdb6966e723cb2e969ad45efb000c0)